### PR TITLE
JNG-5589 multiple actors

### PR DESCRIPTION
**Pull request by noherczeg**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5589" title="JNG-5589" target="_blank">StoryJNG-5589</a>  Support multiple actors in JSL2UI transformation
  </td></table>
  <br />

### CODE REVIEW

File: judo-tatami-jsl-jsl2ui/src/main/epsilon/operations/jsl/data/transferDeclaration.eol [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    // var operations = self.getAllOperations().select(op | not op.hasQueryParameters());
+    /*relations.addAll(operations
+        .select(o | o.input.isDefined())
+        .collect(o | o.input.target)
+        .collect(i | i.getAllRelations().select(r | not r.isQuery)).flatten());
+    relations.addAll(operations
+        .select(o | o.output.isDefined())
+        .collect(o | o.output.target)
+        .collect(o | o.getAllRelations().select(r | not r.isQuery)).flatten());
+    */
```
Comment (bencegelei) [inline judo-tatami-jsl-jsl2ui/src/main/epsilon/operations/jsl/data/transferDeclaration.eol:45]: Is this comment intentional?

File: judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/structure/class.etl [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 
         t.name = s.getFqName() + "::ClassType";
         //t.transferObjectTypeName = s.transferObjectTypeName();
         t.simpleName = s.name;
         //t.setPackageNameTokens(s);
 
-        t.isMapped = s.isMapped();
-        t.isOptional = s.isMapped();
+        // t.isMapped = s.isMapped();
+        // t.isOptional = s.isMapped();
```
Comment (bencegelei) [inline judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/structure/class.etl:15]: Why is it commented?

File: judo-tatami-jsl-jsl2ui/src/main/epsilon/operations/jsl/data/transferDeclaration.eol (hunk 1 above)
Comment (noherczeg) [reply judo-tatami-jsl-jsl2ui/src/main/epsilon/operations/jsl/data/transferDeclaration.eol:45]: Right now, every comment is :D

File: judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/structure/class.etl (hunk 2 above)
Comment (noherczeg) [reply judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/structure/class.etl:15]: Transfer member transformation will come later.

**Review by robertcsakany — APPROVED** [review]

**Review by gaborflorian — APPROVED** [review]

**Pull request merged**